packages/component-library/stories/CivicSandboxCard.story.js

```diff
@@ -0,0 +1,275 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import * as d3 from 'd3';
+import { css } from 'emotion';
+import { storiesOf } from '@storybook/react';
+import { withKnobs, boolean } from '@storybook/addon-knobs';
+import { checkA11y } from '@storybook/addon-a11y';
+import {
+  CivicSandboxCard,
+} from '../src';
+import { wallOfText } from './shared';
+
+const displayName = CivicSandboxCard.displayName || 'CivicSandboxCard';
+
+class LoadData extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      foundation1: null,
+      slide1: null,
+      error: null,
+    };
+  }
+
+  componentDidMount() {
+    let cmp = this;
+    d3.queue()
+      .defer(d3.json, this.props.urls[0])
+      .defer(d3.json, this.props.urls[1])
+      .await((error, foundation1, slide1) => {
+        if (error) {
+          cmp.setState({ error: error });
+        }
+        cmp.setState({ foundation1, slide1 });
+      });
+  }
+
+  render() {
+    if (this.state.foundation1 === null) {
+      return null;
+    }
+    const { foundation1, slide1 } = this.state;
+    return this.props.children(foundation1, slide1);
+  }
+}
+
+const dataURLs = [
+  'https://gist.githubusercontent.com/mendozaline/f78b076ce13a9fd484f6b8a004065a95/raw/ff8bd893ba1890a6f6c20265f720587f9595a9c4/pop.json',
+  'https://gist.githubusercontent.com/mendozaline/b3a75b40c9a60781b6adc77cebb9b400/raw/fa0aa13c75bfcc2fd92ccf1f3cc612af83d5d704/010-grocery.json',
+];
+
+const dashboardComponent = (foundationData1, slideData1) => {
+  if (foundationData1 === null) {
+    return null;
+  }
+
+  const mapboxToken =
+    'pk.eyJ1IjoidGhlbWVuZG96YWxpbmUiLCJhIjoiY2o1aXdoem1vMWtpNDJ3bnpqaGF1bnlhNSJ9.sjTrNKLW9daDBIGvP3_W0w';
+  const mapboxStyle = 'mapbox://styles/hackoregon/cjiazbo185eib2srytwzleplg';
+  const planetColorScheme = [
+    [247, 244, 249, 255],
+    [231, 225, 239, 255],
+    [212, 185, 218, 255],
+    [201, 148, 199, 255],
+    [223, 101, 176, 255],
+    [231, 41, 138, 255],
+    [206, 18, 86, 255],
+    [152, 0, 67, 255],
+    [103, 0, 31, 255],
+  ];
+
+  const populationGetColor = f => {
+    const population = parseFloat(f.properties.total_population);
+    return population < 3000
+      ? planetColorScheme[0]
+      : population < 8000
+        ? planetColorScheme[1]
+        : population < 12000
+          ? planetColorScheme[2]
+          : population < 16000
+            ? planetColorScheme[3]
+            : population < 20000
+              ? planetColorScheme[4]
+              : population < 24000
+                ? planetColorScheme[5]
+                : population < 28000
+                  ? planetColorScheme[6]
+                  : population < 32000
+                    ? planetColorScheme[7]
+                    : planetColorScheme[8];
+  };
+
+  const foundation = {
```

Comment:
perhaps some of these could be split out into a constants file if they are being reused?

Reply:
Yeah, I agree that breaking these out into a constants file is worth it.

Reply:
Discussed and going to wait for the changes on the back-end